depthai_nodes/ml/parsers/classification.py

```diff
@@ -71,16 +108,7 @@ def run(self):
             except dai.MessageQueue.QueueException:
                 break  # Pipeline was stopped
 
-            output_layer_names = output.getAllLayerNames()
-            if len(output_layer_names) != 1:
-                raise ValueError(
-                    f"Expected 1 output layer, got {len(output_layer_names)}."
-                )
-
-            if self.n_classes == 0:
-                raise ValueError("Classes must be provided for classification.")
-
-            scores = output.getTensor(output_layer_names[0], dequantize=True).astype(
+            scores = output.getTensor(self.output_layer_name, dequantize=True).astype(
```

Comment:
`self.output_layer_name` is only defined by calling the `build()` method. We are missing a way to define it manually.